components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -107,10 +113,21 @@
 @PrepareForTest({SCIMGroupHandler.class, IdentityUtil.class, SCIMUserSchemaExtensionBuilder.class,
         SCIMAttributeSchema.class, AttributeMapper.class, ClaimMetadataHandler.class, SCIMCommonUtils.class,
         IdentityTenantUtil.class, AbstractUserStoreManager.class, Group.class, UserCoreUtil.class,
-        ApplicationManagementService.class})
+        ApplicationManagementService.class, RolePermissionManagementService.class, SCIMCommonComponentHolder.class,
+        SCIMUserManager.class})
 @PowerMockIgnore("java.sql.*")
 public class SCIMUserManagerTest extends PowerMockTestCase {
 
+    final static public String USERNAME_LOCAL_CLAIM = "http://wso2.org/claims/username";
```

Comment:
```suggestion
    public static final String USERNAME_LOCAL_CLAIM = "http://wso2.org/claims/username";
```

Fix in other places as well.